common/elasticsearch/src/main/scala/uk/ac/wellcome/elasticsearch/WorksAnalysis.scala

```diff
@@ -46,9 +59,65 @@ object WorksAnalysis {
     tokenizer = "standard",
     tokenFilters = List(
       "lowercase",
-      asciiFoldingTokenFilter.name,
       englishStemmerTokenFilter.name,
-      englishPossessiveStemmerTokenFilter.name),
+      englishPossessiveStemmerTokenFilter.name
+    ),
+    charFilters = Nil
+  )
+
+  val frenchAnalyzer = CustomAnalyzer(
+    "french_analyzer",
+    tokenizer = "standard",
+    tokenFilters = List(
+      "lowercase",
+      frenchStemmerTokenFilter.name
+    ),
+    charFilters = Nil
+  )
```

Comment:
Having a function to create these CustomAnalyzers for each language would reduce the repetition, and make it more obvious if they're all the same or differ in slight ways.

Reply:
They've been removed now...